test/e2e-tests.sh

```diff
@@ -37,6 +37,15 @@ ci_run && {
   (( failed )) && fail_test
 }
 
+tkn() {
+    if [[ -e ./bin/tkn ]];then
+        ./bin/tkn $@
+    else
+        go build github.com/tektoncd/cli/cmd/tkn
```

Comment:
will this create the binary in ./bin? 
I think it command should be `make bin/tkn`